src/App.js

```diff
@@ -28,4 +31,8 @@ class App extends Component {
   }
 }
 
-export default App;
+function mapStateToProps(state) {
+  return state.auth;
+}
+
+export default connect(mapStateToProps)(App);
```

Comment:
@Spetastian How about `export default connect(({auth}) => {auth})(App);` ?

@MartyTheeMartian @Moe-Shoman Right?

Then it's more extensible than having `state` be just `auth` in the reducers.